zng 2.2 now preallocs all the compress memory https://github.com/zlib-ng/zlib-ng/releases/tag/2.2.0
so just bump this by 32k; no idea what the real value is
otherwise we get spammed by 'failed to use prealloc buffer' in the logs (also breaks 2 tests)
--
--- a/src/http/modules/ngx_http_gzip_filter_module.c
+++ b/src/http/modules/ngx_http_gzip_filter_module.c
@@ -527,6 +527,7 @@
 
         ctx->allocated = 8192 + 16 + (1 << (wbits + 2))
                          + 131072 + (1 << (memlevel + 8))
+                         + 32768
                          + 4 * (64 + sizeof(void*));
         ctx->zlib_ng = 1;
     }
